docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
+
+Seven of these eight constants are set to values that provide sufficiently strong anonymity in virtually any realistic scenario. These seven constants rarely if ever need to be modified. Doing so only reduces data quality with little meaningful strengthening of anonymity.
+
+The one anonymization constant that sometimes requires adjustment is `pg_diffix.low_count_min_threshold`. This sets the lower bound for the number of distinct AID values that must be present in a aggregate bucket to avoid suppression. Any buckets with fewer than `pg_diffix.low_count_min_threshold` distinct AIDs will be suppressed. See (How to set suppression threshold)[#how-to-set-suppression-threshold] for further guidance.
+
+Default value is 3. Minimum allowed setting is 2.
+
+#### Other seven settings
+
+`pg_diffix.noise_layer_sd` - Standard deviation for each noise layer added to aggregates. Default value is 1.0. Minimum allowed setting is 1.0.
+
+`pg_diffix.low_count_mean_gap` - The number of standard deviations between the lower bound `pg_diffix.low_count_min_threshold` and the mean of the
+low count filter threshold. Default value is 2.0. Minimum allowed setting is 2.0.
 
-At access levels other than `direct`, various data and features built into PostgreSQL are restricted. Among others:
+`pg_diffix.low_count_layer_sd` - The standard deviation for each noise layer used when calculating the low count filter threshold. Default value is 1.0. Minimum allowed setting is 1.0.
+
+`pg_diffix.outlier_count_min` - Default value is 1. Must not be greater than `outlier_count_max`. Minimum allowed setting is 1.
+
+`pg_diffix.outlier_count_max` - Default value is 2. Must not be smaller than `outlier_count_min`. Minimum allowed setting is 2.
+
+`pg_diffix.top_count_min` - Default value is 3. Must not be greater than `top_count_max`. Minimum allowed setting is 2.
+
+`pg_diffix.top_count_max` - Default value is 4. Must not be smaller than `top_count_min`. Minimum allowed setting is 3.
+
+**NOTE** The outlier interval `(outlier_count_min, outlier_count_max)` must not be wider than the `(top_count_min, top_count_max)` interval.
+
+### Anonymization reporting settings
+
+`pg_diffix.compute_suppress_bin` - If `True`, the first row in the query result contains the suppress bin (if any). This
+provides the combined anonymized count of all the bins that were suppressed. The suppress bin shows
+all column values as `NULL` (`*` for text-typed columns, customizable via `pg_diffix.text_label_for_suppress_bin`). Note
+that the suppress bin may itself be suppressed.
+
+`pg_diffix.text_label_for_suppress_bin` - The value to use for the text-typed grouping labels in the suppress bin row.
+Default value is `*`.
+
+## Restricted features and extensions
+
+**TODO:** I think this kind of information is better put in the notebook tutorial?
```

Comment:
It mostly discusses utility statements and provides a forewarning to know what is blocked. The notebook itself will highlight query features.

Reply:
I'm going to ask one of you to make a pass at this after merge. It seems incomplete or something...